cli/internal/cmd/iamcreate.go

```diff
@@ -396,6 +396,10 @@ func (c *awsIAMCreator) parseFlagsAndSetupConfig(cmd *cobra.Command, flags iamFl
 	if err != nil {
 		return iamFlags{}, fmt.Errorf("parsing zone string: %w", err)
 	}
+	// check that zone has availability zone suffix
+	if !strings.HasSuffix(zone, "a") && !strings.HasSuffix(zone, "b") && !strings.HasSuffix(zone, "c") {
+		return iamFlags{}, fmt.Errorf("missing availability zone in %s flag. Availability zone is needed to support the default attestation variant SEV-SNP. please refer to our docs and https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-availability-zones", zone)
```

Comment:
Also I think we had some "policy" on personal addressing of users (e.g. "please refer"), but I'm not entirely sure anymore. @thomasten should know best

Reply:
We follow the Microsoft Style Guide. (You may add this to the dev-docs). Rules that apply here:
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/please (related: https://learn.microsoft.com/en-us/style-guide/top-10-tips-style-voice#be-brief)
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/we